python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -430,7 +449,12 @@ def process_graph(args):
         edges = edges1
         sys_tracker.check('Add reverse edges')
     g = dgl.heterograph(edges, num_nodes_dict=num_nodes)
-    print(g)
+    logging.info("The graph has %d node types and %d edge types.",
+                 len(g.ntypes), len(g.etypes))
+    for ntype in g.ntypes:
+        logging.info("Node type %s has %d nodes.", ntype, g.number_of_nodes(ntype))
+    for etype in g.canonical_etypes:
+        logging.info("Edge type %s has %d edges.", str(etype), g.number_of_edges(etype))
```

Comment:
What about train/val/test info?

Reply:
print out train/val/test split?

Reply:
then we should print out features as well.